01-syntax/04-high-order-functions/mother-child-age/ageDifference.js

```diff
@@ -1,22 +1,112 @@
 /* global ancestry */
 
-var byName = {};
+var ANCESTRY_FILE = '[\n  ' + [
```

Comment:
Why you don't use `ancestry` variable ? 
It should be available because  of [this](https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/mother-child-age/index.html#L6) and [that](https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/data/ancestry.js)

Also, I see that you are not running `npm run lint` before commits